src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -154,6 +156,13 @@ export class CephfsSubvolumeListComponent extends CdForm implements OnInit, OnCh
         permission: 'delete',
         icon: Icons.destroy,
         click: () => this.removeSubVolumeModal()
+      },
+      {
+        name: this.actionLabels.ATTACH,
+        permission: 'read',
+        icon: Icons.bars,
+        disable: () => !this.selection?.hasSelection,
+        click: () => this.showAttachInfo()
```

Comment:
same here, `Remove` should be the last action!

Reply:
done